src/transformers/index.ts

```diff
@@ -40,7 +40,9 @@ import { notUndefined } from '~utils/arrays';
 //   return Array.from(roleSet);
 // };
 
-const convertRolesToArray = (rolesInDomain?: ColonyRoleFragment | null) =>
+export const convertRolesToArray = (
+  rolesInDomain?: PickStartsWith<ColonyRoleFragment, 'role_'> | null,
```

Comment:
I can live with this, but I much prefer specific picks than generics, it's more readable and self documenting